src/PlayerManager.cpp

```diff
@@ -601,10 +609,16 @@ void PlayerManager::exitPlayerVehicle(CNSocket* sock, CNPacketData* data) {
     INITSTRUCT(sP_FE2CL_PC_VEHICLE_OFF_SUCC, response);
     PlayerView plrv = PlayerManager::players[sock];
 
-    // send to other players
-    //for (CNSocket* otherSock : plrv.viewable) {
-    //    otherSock->sendPacket((void*)&response, P_FE2CL_PC_VEHICLE_OFF_SUCC, sizeof(sP_FE2CL_PC_VEHICLE_OFF_SUCC));
-    //}
+    //send to other players
+    INITSTRUCT(sP_FE2CL_PC_EQUIP_CHANGE, pkt);
+    pkt.EquipSlotItem.iType = 1;
+    pkt.iEquipSlotNum = 8;
+    for (CNSocket* otherSock : plrv.viewable) {
+        otherSock->sendPacket((void*)&pkt, P_FE2CL_PC_VEHICLE_ON_SUCC, sizeof(sP_FE2CL_PC_EQUIP_CHANGE));
```

Comment:
Same problem here; mismatched packet id and struct type.

Reply:
Same comment as above.